src/components/Form/CheckAclPermsDocContainerForm.jsx

```diff
@@ -28,8 +28,8 @@ import FormSection from './FormSection';
 const CheckAclPermsDocContainerForm = () => {
   const { session } = useSession();
   const { state, dispatch } = useStatusNotification();
-  const { selectedUser } = useContext(SelectUserContext);
-  const [username, setUsername] = useState('');
+  const { selectedUser } = useContext(SelectedUserContext);
+  const [username, setUsername] = useState(selectedUser.username);
```

Comment:
I see selectedUser defaults to the user's own podUrl if unselected. Think it's a great idea. It'll simply a lot of the functions we have to just use podUrl.

For the text displayed in "Client selected", think we could have it show "Personal Pod" instead of the podUrl. Think we could include an entry to the object so that it'll be `{podUrl, person: "Personal Pod" }` for the default selectedUser object or something of the like.

Reply:
fixed